brainlit/utils/swc.py

```diff
@@ -512,3 +522,51 @@ def swc2skeleton(swc_file, origin=None):
     skel.vertex_color[:, :] = color
 
     return skel
+
+
+def swc2skeleton_benchmarking(swc_file, origin=None):
```

Comment:
it looks like the only difference between this method and `swc2skeleton` method is the way you get the ID and `origin` not being checked before subtraction. Seems reasonable to just update the existing method, why make a new one?